app/config/index.js

```diff
@@ -1,38 +1,29 @@
-const Joi = require('joi')
-const dbConfig = require('./db')
-const messageQueueConfig = require('./message-queue')
+import joi from 'joi'
+import dbConfig from './db.js'
+import messageQueueConfig from './message-queue.js'
 
-const schema = Joi.object({
-  port: Joi.number(),
-  env: Joi.string().valid('development', 'test', 'production').default('development'),
-  isDev: Joi.boolean().default(false),
-  termsAndConditionsUrl: Joi.string().default('#'),
-  applyServiceUri: Joi.string().default('#'),
-  claimServiceUri: Joi.string().default('#'),
-  endemics: Joi.object({
-    enabled: Joi.boolean().default(false)
-  })
+const schema = joi.object({
+  port: joi.number(),
+  env: joi.string().valid('development', 'test', 'production').default('development'),
+  isDev: joi.boolean().default(false),
+  termsAndConditionsUrl: joi.string().default('#'),
+  applyServiceUri: joi.string().default('#'),
+  claimServiceUri: joi.string().default('#')
 })
 
-const config = {
-  port: process.env.PORT,
+const baseConfig = {
+  port: process.env.PORT ? Number(process.env.PORT) : undefined,
   env: process.env.NODE_ENV,
   isDev: process.env.NODE_ENV === 'development',
   termsAndConditionsUrl: process.env.TERMS_AND_CONDITIONS_URL,
```

Comment:
See comment above but the Urls aren;t needed

Reply:
Cool! Removed them.